src/gsi/hybrid_ensemble_isotropic.F90

```diff
@@ -3909,7 +4011,8 @@ subroutine ckgcov_a_en_new_factorization_ad(ig,z,a_en)
   use constants, only: zero
   use gridmod, only: regional
   use hybrid_ensemble_parameters, only: n_ens,grd_loc
-  use hybrid_ensemble_parameters, only: nval_lenz_en
+  use hybrid_ensemble_parameters, only: nval_loc_en
```

Comment:
@shoyokota  As we discussed,  nval_loc_en should be used as the local (private ) variable to this module. 
It will not be used outside this module and will not have any non-local impacts.

Reply:
Even in this coding, nval_loc_en is used only in this module substantially, but I changed it to the strictly local variable just in case.

Reply:
@shoyokota  "nval_loc_en is used only in this module substantially"  is why  it should be declared as local variable.